types/ref-array-di/tslint.json

```diff
@@ -0,0 +1,79 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "adjacent-overload-signatures": false,
```

Comment:
Do you really need to disable all of these lint rules? If so, can you document why?

Reply:
@uniqueiniquity This was included as part of this PR https://github.com/DefinitelyTyped/DefinitelyTyped/pull/21009
which was introduced because of this  https://github.com/DefinitelyTyped/DefinitelyTyped/pull/20820#issuecomment-339363329